### DISCUSSION TITLE
FailedAttachVolume on Second Node

### DISCUSSION THREAD

**Discussion by northboys** [longhorn/longhorn#3475] [Q&A]
Hi, I have added new node on longhorn and trying to deploy pod and pvc on new node, but i got error like this
<img width="1680" alt="Screen Shot 2021-12-30 at 08 18 55" src="https://user-images.githubusercontent.com/38155944/147714521-36302918-1c60-430a-a3a1-0dea913f9dfb.png">

**Comment by jenting** [comment]
Could you check the open-iscsi package and the iscsid service be enabled?
